src-vue/overlays/BootingOverlay.vue

```diff
@@ -75,7 +78,32 @@ const config = useConfig();
 const draggable = Vue.reactive(new Draggable());
 
 const syncErrorType = Vue.ref<string | null>(null);
+const toRestore = localStorage.getItem('ConfigRestore');
+localStorage.removeItem('ConfigRestore');
 
+async function applyRestoredServer(details: string) {
+  try {
+    const restoreData = JSON.parse(details) as Record<keyof IConfig, string>;
+    if (restoreData.serverDetails) {
+      const parseResult = ConfigServerDetailsSchema.safeParse(JsonExt.parse(restoreData.serverDetails));
+      console.log(parseResult);
+      if (!parseResult.success) return;
+
+      const serverDetails = parseResult.data;
+      await SSH.tryConnection(serverDetails, config.security.sshPrivateKey);
+      config.isServerUpToDate = false;
+      config.serverDetails = serverDetails;
+      await config.save();
+      await SSH.closeConnection();
```

Comment:
SSH.closeConnection() does not close the connection created by SSH.tryConnection().

Reply:
Copied this straight from ServerConfigureeOverlay. Should it be removed in both?

Reply:
I'm not even sure we want to close the connection since it's hopefully about to be used.

Reply:
Holding now in try connection and removed the close

Reply:
It might be clearer if we remove SSH.closeConnection and add a SSH.getConnection. Then ServerConfigureOverlay could call `const previousConnection = SSH.getConnection()` before SSH.tryConnection() on new server. If new server works then it can run `previousConnection.close()`.